vortex-datafusion/src/persistent/source.rs

```diff
@@ -176,9 +175,6 @@ impl FileSource for VortexSource {
             DisplayFormatType::Default | DisplayFormatType::Verbose => {
                 if let Some(predicate) = &self.predicate {
                     write!(f, ", predicate={predicate}")?;
-                    if let Some((pruning_predicate, _)) = pruning_expr(predicate) {
```

Comment:
brutal... if you get rid of this branch then all the cases are the same

Reply:
merged them

Reply:
I guess for pretty printing cases you want to not print `, ` tree case